From 6b73a21f7cb1c8d0e6011052054bb78b28dc5bc7 Mon Sep 17 00:00:00 2001
From: Dongjin Kim <tobetter@gmail.com>
Date: Sat, 2 May 2015 18:07:07 -0700
Subject: [PATCH] ODROID-C/dts: Fix side-effect issue due to CEC changes

This patch is to revert the changes due to CEC except CEC specific node.

Change-Id: Ifd97c3064ca42d39319dd237340b5ac2c051cdff
Signed-off-by: Dongjin Kim <tobetter@gmail.com>
---
 arch/arm/boot/dts/meson8b_odroidc.dts | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/boot/dts/meson8b_odroidc.dts b/arch/arm/boot/dts/meson8b_odroidc.dts
index 0609574..6ed7ac5 100755
--- a/arch/arm/boot/dts/meson8b_odroidc.dts
+++ b/arch/arm/boot/dts/meson8b_odroidc.dts
@@ -99,7 +99,7 @@
                 vmode = <0>; /**0:VMODE_720P 1:VMODE_LCD  2:VMODE_LVDS_1080P 3:VMODE_1080P*/
                 scale_mode = <1>; /*0:default 1:new*/
                 4k2k_fb = <0>;
-                display_size_default = <1280 720 1280 1440 32>; //1920*1080*4*3 = 0x17BB000
+                display_size_default = <1920 1080 1920 3240 32>; //1920*1080*4*3 = 0x17BB000
         };
 
         mesonstream{
@@ -263,11 +263,11 @@
             status = "okay";
             port = <5>;          /**0:sdio_a, 1:sdio_b, 2:sdio_c, 3:sdhc_a, 4:sdhc_b, 5:sdhc_c */
             pinname = "emmc";
-            ocr_avail = <0x00200080>;          /**VDD voltage 3.3 ~ 3.4 */
+            ocr_avail = <0x200000>;
             caps = "MMC_CAP_8_BIT_DATA","MMC_CAP_MMC_HIGHSPEED","MMC_CAP_SD_HIGHSPEED", "MMC_CAP_NONREMOVABLE","MMC_CAP_ERASE", "MMC_CAP_HW_RESET"; // MMC_CAP_NEEDS_POLL -- for detect, MMC_CAP_NONREMOVABLE -- for eMMC/TSD
             caps2 = "MMC_CAP2_HS200_1_8V_SDR", "MMC_CAP2_BOOTPART_NOACC";
 	        f_min = <300000>;
-            f_max = <100000000>;
+            f_max = <150000000>;
             max_req_size = <0x20000>;          /**128KB*/
             gpio_dat3 = "BOOT_3";
             card_type = <1>; /* 0:unknown, 1:mmc card(include eMMC), 2:sd card(include tSD), 3:sdio device(ie:sdio-wifi), 4:SD combo (IO+mem) card, 5:NON sdio device(means sd/mmc card), other:reserved */
-- 
2.9.5

